client/disputes/index.tsx

```diff
@@ -345,16 +345,17 @@ export const DisputesList = (): JSX.Element => {
 	const downloadable = !! rows.length;
 
 	const onDownload = async () => {
-		recordEvent( 'wcpay_disputes_download', {
-			exported_disputes: rows.length,
-			total_disputes: disputesSummary.count,
```

Comment:
We're losing the "total" count for disputes at least. 

If it's possible to include, let's include it.

- `exported_row_count` - the number of rows in the export
- `total_row_count` - the total number of rows available

Then we will be able to see filtered/partial exports in the tracks data.

Reply:
The total_disputes earlier actually indicated the total exported rows. I checked the API response and found there is no easy way currently to get the total number of disputes / transactions / payouts that is there for the user. The API only returns the total rows for the query.

Reply:
Thanks for checking – in that case let's skip the `total` prop 👍🏻